Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+
+- Outputs:
+  - SATRE specification V1 and [associated technical paper](https://zenodo.org/records/10053383)
+  - Internal DARE report, link will be added and published openly
+  - [User report](https://zenodo.org/records/10066800): characterises users, with a wider notion than we started of what users are. Lays the fundation for futher usability work, going beyong technicla features and into training and documentation
+  - UK TRE Community - SATRE WG: the ongoing work and evolution of the specification is now a working group within the Uk TRE Community
+
+### Documentation management
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/32
+
+Contributes to:
+- Identifying, co-creating and supporting a TRE standard used across TRE infrastructures
+- Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+- Identifying and documenting everything that can be openly documented
+
+#### Goal
+Comprehensive and clear documentation for the DSH, SATRE & TRESA will ensure open, reproducible outputs from this project.
+- Ensure all relevant information is captured in documentation
+- Test accessibility and discoverability of docs with relevant groups
+- Iterate documentation in line with wider project work (e.g. TRESA processes, DSH updates)
+- Determine what can/can't be documented (e.g. from an IG perspective).
+
+#### Definition of Done
+When funding ends for the project and we have openly documented everything that we feel we can
+
+#### Progress
+Documentation management have not been a story actively worked in, yet some processes have been updated and documented and needs identified.
+While the story goes beyond Production processes it is worth noting that those have been handed over to TRESA, who are already suggesting and appliying changes.
+
+### SATRE: stakeholder engagement and community buy out
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/66
+
+#### Goal
+Ensure that institutions evaluate themselves against SATRE and that the momentum is maintained between funded phases
+
+At the end of the funded phase of SATRE there was a growing community interest, with institutions and stakeholders affirming they would evaluate themselves against it and contribute feedback.
+
+Without ongoing resources the necessary support to ensure that happens cannot be provided and SATRE may end up not being adopted.
+
+#### Definition of Done
+There are a number of self evlauaitons completed, there is feedback on the spec repository and, ideally, there is an active WG within the TRE Community continuing to work on it.
+
+#### Progress
+Work is folded into the [UK TRE Community](https://github.com/alan-turing-institute/data-safe-haven-team/issues/52), already SATRE is formally becoming a UK TRE Community WG.
+
+## Community building
+Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
```

Comment:
```suggestion
Creating resources for all stakeholders (inc. members of the public) to engage in the TRE conversation
```

I don't need to die on this hill but I do think "citizens" has a lot of baggage related to immigration and _citizenship_ that we probably don't need to introduce here!